people/joboet.toml

```
@@ -0,0 +1,4 @@
+name = "joboet"
```

@joboet The tool picked your github username by default for the `name` field, but this can be anything you like (e.g. your full name or another nickname). You can also add an email address.

---

All right, then let's use my full name:
```suggestion
name = "Jonas Böttiger"
```